docs/config.ini

```diff
@@ -30,7 +34,6 @@ guard=onebot.plugins.acl.user_based_policy
 identify_by = mask
 
 [onebot.plugins.acl]
-# Pre-seed acl
```

Comment:
I founded pre-seeding acl wasn't necessary when the plugin was added to the includes list